_src/app/catch/Catch.js

```diff
@@ -520,9 +520,18 @@ define([
                 if (count === 1) {
                     expandedRows.push(nextRow);
                 } else {
+                    const weight = nextRow[FN.WEIGHT];
+                    let averageWeight;
+                    if (weight) {
+                        averageWeight = nextRow[FN.WEIGHT] / count;
```

Comment:
i always remember the js talk about math in js when I see division. Isn't there a recommendation to multiply by 100 and divide by 100 after the operation to get the precision correct?